### DISCUSSION TITLE
When the rumble/ vibration will be added to xbox and PS4 controllers?

### DISCUSSION THREAD

**Discussion by Khaledgee** [ndeadly/MissionControl#219] [General]
When the rumble/ vibration will be added to Xbox and PS4 controllers? And what shall we expect in the upcoming release?

**Comment by ndeadly** [comment]
I plan to include rumble for those controllers in the next release. The update is pretty much ready, I just need to fix some things that broke on older firmwares first. Unfortunately I've been pretty busy in my personal life to work on the project much lately.
